.github/workflows/ci.yml

```diff
@@ -11,7 +11,9 @@ on:
   merge_group:
 
 env:
+  PRIMARY_ESLINT_VERSION: '^8.0.0'
   PRIMARY_NODE_VERSION: 18
+  PRIMARY_TYPESCRIPT_VERSION: '~4.8.0'
```

Comment:
side note that i hate that this is yet another thing for us to remember to keep in sync.